ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.21__multi-nightly.yaml

```diff
@@ -1466,8 +1466,61 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-azure-ipi-multidisk
-- as: azure-ipi-nat-gateway-single-zone-amd-f28-destructive
-  cron: 45 22 9 * *
+- as: azure-ipi-nat-gateway-mzone-mini-perm-amd-f28-destructive
+  cron: 52 15 20 * *
+  steps:
+    cluster_profile: azure-qe
+    env:
+      AZURE_INSTALL_USE_MINIMAL_PERMISSIONS: "yes"
+      BASE_DOMAIN: qe.azure.devcluster.openshift.com
+      CUSTOM_AZURE_REGION: centralus
+      OUTBOUND_TYPE: NATGatewayMultiZone
+    test:
+    - chain: openshift-e2e-test-qe-destructive
+    workflow: cucushift-installer-rehearse-azure-ipi-nat-gateway
+- as: azure-ipi-nat-gateway-mzone-perm-mini-arm-f7
+  cron: 29 14 6,13,22,29 * *
+  steps:
+    cluster_profile: azure-qe
+    env:
+      AZURE_INSTALL_USE_MINIMAL_PERMISSIONS: "yes"
+      BASE_DOMAIN: qe.azure.devcluster.openshift.com
+      COMPUTE_NODE_TYPE: Standard_D4ps_v5
+      CUSTOM_AZURE_REGION: centralus
+      OCP_ARCH: arm64
+      OUTBOUND_TYPE: NATGatewayMultiZone
+    test:
+    - chain: openshift-e2e-test-qe
+    workflow: cucushift-installer-rehearse-azure-ipi-nat-gateway
+- as: azure-ipi-nat-gateway-mzone-byo-subnets-amd-f28-destructive
+  cron: 56 14 14 * *
+  steps:
+    cluster_profile: azure-qe
+    env:
+      AZURE_COMPUTE_SUBNETS_NUMBER: "3"
```

Comment:
Does the number of subnets be consistent with the number of available zones? If so, should we directly define `AZURE_COMPUTE_SUBNETS_NUMBER` by getting the number of zones in the script?

Reply:
When specifying outboundType to NATGatewayMultiZone without subnets setting in install-config.yaml, installer will create compute sunbets, the number is matched with available zones.

Here `AZURE_COMPUTE_SUBNETS_NUMBER` just defines the byo compute subnets number. Maybe I can rename this ENV more clearly, such as "AZURE_BYO_COMPUTE_SUBNETS_NUMBER"